addons/binding/org.openhab.binding.snapcast/src/main/java/org/openhab/binding/snapcast/internal/data/Client.java

```diff
@@ -0,0 +1,650 @@
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.snapcast.internal.data;
+
+import com.google.gson.annotations.SerializedName;
+
+/**
+ * The {@link Client} is a data structure for the api communication.
+ * It represents a snapcast client.
+ *
+ * @author Steffen Brandemann - Initial contribution
+ */
+public class Client implements Identifiable {
+
+    @SerializedName("id")
+    private String id;
+
+    @SerializedName("config")
+    private ClientConfig config;
+
+    @SerializedName("connected")
+    private Boolean connected;
+
+    @SerializedName("host")
+    private Host host;
+
+    @SerializedName("lastSeen")
+    private LastSeen lastSeen;
+
+    @SerializedName("snapclient")
+    private Snapclient snapclient;
+
+    public Client() {
```

Comment:
You don't have to add this construct, imho its overly verbose.
If there is no constructor Java will automatically add a default constructor (public with no-arguments).